spring-cloud-open-service-broker-core/src/main/java/org/springframework/cloud/servicebroker/controller/CatalogController.java

```diff
@@ -55,7 +55,15 @@ public CatalogController(CatalogService service) {
 	@GetMapping({"/v2/catalog", "{platformInstanceId}/v2/catalog"})
 	public Mono<Catalog> getCatalog() {
 		return catalogService.getCatalog()
-				.doOnRequest(v -> LOG.debug("Retrieving catalog"));
+				.doOnRequest(v -> LOG.info("Retrieving catalog"))
+				.doOnSuccess(catalog -> {
+					LOG.info("Success retrieving catalog");
+					LOG.debug("catalog={}", catalog);
+				})
+				.doOnError(e -> {
+					LOG.error("Error retrieving catalog");
+					LOG.debug(e.getMessage(), e);
```

Comment:
Are you trying to suppress noise in the logs? 

```
LOG.error("Error retrieving catalog");
LOG.debug(e.getMessage(), e);
```

it seems a bit odd that we have to turn on debug logging to see the reason for the failure, but maybe this is sufficient?

Reply:
I was copying the same pattern as the others in an effort to avoid logging sensitive info, but you make a good point. Error logging should contain the error info. I'll combine the error loggings.